docs/themes/thxvscode/assets/scss/includes/_docs.scss

```diff
@@ -274,15 +290,20 @@
 
 /* Main Nav active */
 #docs-navbar > .nav > .active > a,
-#docs-navbar > .nav > .active .active > a {
-	color: $link-color !important;
+#docs-navbar > .nav > .active .active > a,
+#docs-navbar > .nav > .active > div,
+#docs-navbar > .nav > .active .active > div {
+    color: $link-color !important;
 }
 
+#docs-navbar > .nav > .active > div:not(.area),
```

Comment:
Nit: for consistency, should these `div` entries be listed after the `a` entries?